framework/auth/oauth_scopes.py

```diff
@@ -71,6 +77,9 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ,)
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
 
+    USERS_FULL_READ = USERS_READ + (CoreScopes.USERS_ADDON_READ,)
```

Comment:
I'm working on scopes right now to try to make them usable.

Also, email addresses are not currently available through the API. But changing your twitter handle might not give the app the rights to connect an addon.